packages/mui-joy/src/Tooltip/Tooltip.tsx

```diff
@@ -423,7 +415,7 @@ const Tooltip = React.forwardRef(function Tooltip(inProps, ref) {
   }, [handleClose, open]);
 
   const handleUseRef = useForkRef(setChildNode, ref);
-  const handleFocusRef = useForkRef<Element>(focusVisibleRef, handleUseRef);
+  const handleFocusRef = useForkRef<Element>(handleUseRef);
```

Comment:
I think this step is not needed. Could we pass `handleUseRef` directly to `handleRef` instead?

Reply:
Removed 👍🏼